pydantic_factories/extensions/beanie_odm.py

```diff
@@ -47,7 +47,7 @@ def is_ignored_type(cls, value: Any) -> bool:
         return super().is_ignored_type(value=value) or value is PydanticObjectId
 
     @classmethod
-    def get_field_value(cls, model_field: ModelField) -> Any:
+    def get_field_value(cls, model_field: ModelField, field_kwargs: Any = None) -> Any:
```

Comment:
```suggestion
    def get_field_value(cls, model_field: ModelField, **field_kwargs: Dict[str, Any]) -> Any:
```

Reply:
This parameter type is in fact: `Union[dict, list, None]` because for list of PydanticModels (pets in the example below) the value will be a list of dict (or a list of Pydantic models depending in how the user uses the factory).

So, to avoid any misunderstanding I changed the variable name to `field_parameters: Union[dict, list, None]`.

```python
class Pet(BaseModel):
    name: str
    age: int


class Person(BaseModel):
    name: str
    pets: list[Pet]
    age: int
```

Reply:
ok, good.